src/main/java/dev/bitbite/logging/Log.java

```diff
@@ -34,6 +36,24 @@ protected String format(LogLevel logLevel, Category category, String message, Ha
 		return this.replaceDynamicElements(logLevel, category, message, modify);
 	}
 	
+	/**
+	 * Formats the template String defined in {@link LogProperties} accordingly with actual information.
+	 * @param logLevel for the formatting
+	 * @param message for the replacement
+	 * @param replacements HashMap with user defined templates with their replacement information
+	 * @return the formatted String
+	 */
+	protected String format(LogLevel logLevel, String message, HashMap<String, String> replacements) {
+		String modify = this.properties.getLogTemplate();
+		for(var replacement : replacements.entrySet()) {
+			modify = modify.replace(replacement.getKey(), (!replacement.getValue().contains("\u001b[") || this.properties.usesAnsi()) ? replacement.getValue() : "");
```

Comment:
When the replacement contains an ansi escape sequence, only the sequence should be removed, not the rest

Reply:
escapes and displayed content are seperated, so only the escape is removed.